tests/test_magnetic_fields.py

```diff
@@ -60,6 +60,85 @@ def test_basic_fields(self):
             (tfield + vfield - pfield)([1, 0, 0.1]), [[0.4, 2, 1]]
         )
 
+    @pytest.mark.unit
+    def test_combined_fields(self):
+        """Tests for sum/scaled fields."""
+        tfield = ToroidalMagneticField(2, 1)
+        vfield = VerticalMagneticField(1)
+        pfield = PoloidalMagneticField(2, 1, 2)
```

Comment:
What are these indices? What is this test doing? Can you add more info in the docstring?

Reply:
they are not indices, they are the arguments to the init of each. e.g. the B0 and R0 for the `ToroidalMagneticField`, the B0 for `VerticalMagneticField`